client/src/app/editor/file-outputs/file-outputs-table.component.scss

```diff
@@ -0,0 +1,53 @@
+mat-cell {
+  mat-icon {
+    vertical-align: -25%;
+    color: #607d8b;
+    font-size: 1.7em;
+    width: initial;
+    height: initial;
+  }
+
+  a mat-icon {
+    vertical-align: -38%;
+  }
+}
+
+mat-row:last-of-type {
+  border-bottom: 0;
+}
+
+[mat-icon-button][disabled] {
+  mat-icon {
+    opacity: 0.2;
+  }
+}
+
+.mat-column-name {
+  white-space: nowrap;
+  text-overflow: ellipsis;
+
+  .text-ellipsis {
+    display: inline-block;
+    overflow: hidden;
+  }
+}
+
+.mat-column-actions {
+  display: flex;
+}
+
+mat-progress-spinner {
+  width: 16px !important;
+  height: 16px !important;
+}
+
+.message {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-top: 1rem;
+
+  /deep/ mat-progress-spinner {
```

Comment:
I think we wanna use `:ng-deep` here instead.

Reply:
Already fixed.

Reply:
No no. Read my comment again :D

Reply:
`:ng-deep` is deprecated in favor of `/deep/` no?

Reply:
Exactly this ^

So we just stick to :ng-deep